cmake/gtest/CMakeLists.txt.gtest.in

```diff
@@ -6,7 +6,7 @@ project(googletest-download NONE)
 include(ExternalProject)
 ExternalProject_Add(googletest
         GIT_REPOSITORY https://github.com/google/googletest.git
-        GIT_TAG master
+        GIT_TAG release-1.11.0
```

Comment:
I don't understand this change.

Reply:
The main branch of googletest has been renamed from 'master' to 'main' so a switch from the first to the second was necessary to download the project. I took the opportunity to pin a version rather than relying on a branch that may change without notice. Do you want me to switch back to the branch 'main'?